### DISCUSSION TITLE
I can not connect to ssh server

### DISCUSSION THREAD

**Discussion by iranimij** [wardenenv/warden#693] [Help & Support]
### Version of Warden

in-dev

### Operating System and Installation Method

Linux ubuntu 22

### Describe the Bug

I can not connect php storm to the database.
The issue is that It can not connect to tunnel server and I receive "Authentication error"

I have check the tunnel server : 
ssh user@tunnel.warden.test and it returns : 
`user@127.0.0.1: Permission denied (publickey,keyboard-interactive).
`

I have access to the database from my browser it means the I am sure that the database has been imported correctly.

Would you please give me a way to regenerate the tunnel keys of warden?

### To Reproduce

I can not reproduce

**Comment by manuelson** [comment]
Hello @iranimij , you must first connect to port 2222 and use the credentials (id_rsa) of warden .warden/tunnel

Please check the page https://docs.warden.dev/configuration/database.html
